### PR TITLE
Update Simplified Chinese localization

### PR DESCRIPTION
**Pull request by Jerry23011**
### Changes
- Add new translations

@nixzhu would you like to review them 😄

### CODE REVIEW

File: IceCubesApp/Resources/Localization/zh-Hans.lproj/Localizable.strings [hunk 1]
```diff
@@ -180,8 +180,8 @@
 "settings.swipeactions.status.trailing.left" = "左";
 "settings.swipeactions.status.trailing.right" = "右";
 "settings.swipeactions.status" = "嘟文";
-"settings.swipeactions.status.use-theme-colors" = "Use theme colors instead of default colors";
-"settings.swipeactions.status.icon-style" = "Icon style";
+"settings.swipeactions.status.use-theme-colors" = "使用应用主题色";
```
Comment (nixzhu) [inline IceCubesApp/Resources/Localization/zh-Hans.lproj/Localizable.strings:183]: 使用主题色而不是默认色

File: IceCubesApp/Resources/Localization/zh-Hans.lproj/Localizable.strings [hunk 2]
```diff
@@ -383,8 +383,8 @@
 "status.editor.error.upload" = "上传错误";
 "status.editor.language-select.navigation-title" = "选择语言";
 "status.editor.language-select.recently-used" = "最近使用";
-"status.editor.language-select.confirmation.detected-%@" = "Post in %@ (Detected language)";
-"status.editor.language-select.confirmation.selected-%@" = "Post in %@ (Selected language)";
+"status.editor.language-select.confirmation.detected-%@" = "嘟文检测为%@";
```
Comment (nixzhu) [inline IceCubesApp/Resources/Localization/zh-Hans.lproj/Localizable.strings:386]: 推荐加个空格：`嘟文检测为 %@`

File: IceCubesApp/Resources/Localization/zh-Hans.lproj/Localizable.strings [hunk 3]
```diff
@@ -383,8 +383,8 @@
...
+"status.editor.language-select.confirmation.selected-%@" = "使用用%@发送";
```
Comment (nixzhu) [inline IceCubesApp/Resources/Localization/zh-Hans.lproj/Localizable.strings:387]: 使用 %@ 发送

**Review by nixzhu — CHANGES_REQUESTED** [review]

**Comment by Jerry23011** [comment]
都OK啦

**Comment by nixzhu** [comment]
LGTM

**Pull request merged**